src/styles/blocks/header.scss

```diff
@@ -0,0 +1,69 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05); 
+  padding: 0 50px;
+
+  &__link {
+    display: flex;
+    align-items: center;
+  }
+
+  &__nav {
+    display: flex;
+    align-items: center;
+    height: 60px;
+
+    &-list {
+      display: flex;
+      list-style: none;
+      padding: 0;
+      height: 100%;
+    }
+
+    &-item {
+      position: relative;
+      height: 100%;
+      margin: 0 10px;
+    }
+
+    &-link {
+      display: flex;
+      align-items: center;
+      text-decoration: none;
+      height: 100%;
+      font-size: 12px;
```

Comment:
The font-size value does not adhere to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), which suggest using rem units for font-size to improve accessibility and maintainability of the code.